src/Json.Schema.ToDotNet/ClassGenerator.cs

```diff
@@ -162,9 +166,15 @@ public override void AddMembers()
                 members.Add(GenerateValueGetHashCodeMethod());
             }
 
+            if (_generateComparers)
+            {
+                Usings.Add("System.Collections.Generic");
```

Comment:
>Usings.Add [](http://example.com/codeflow?start=16&length=10)

does Usings.Add ignore duplicates when both _generateEqualityComparers and _generateComparers true

Reply:
good point, the `Usings`  type is `HashSet<string>`, and it does not add duplicated entries.